tests/e2e/package.json

```diff
@@ -5,22 +5,28 @@
   "main": "dist/index.js",
   "scripts": {
     "lint": "tslint --fix -p .",
-    "tsc": "tsc -p .",
+    "tsc": "./configs/sh-scripts/generateIndex.sh && tsc -p .",
     "cleanup-docker": "if [ $(docker ps -a | grep -c selenium-e2e) -gt 0 ]; then docker rm -f $(docker ps --filter \"name=selenium-e2e\" -aq); fi;",
     "test-docker": "npm run cleanup-docker && docker run -it --shm-size=2g -p 5920:5920 --name selenium-e2e -e TS_SELENIUM_BASE_URL=$TS_SELENIUM_BASE_URL eclipse/che-e2e:nightly",
     "test-docker-mount-e2e": "npm run cleanup-docker && docker run -it --shm-size=2g -p 5920:5920 --name selenium-e2e -e TS_SELENIUM_BASE_URL=$TS_SELENIUM_BASE_URL -v $(pwd):/tmp/e2e:Z eclipse/che-e2e:nightly",
-    "test-all-devfiles": "./configs/sh-scripts/generateIndex.sh && ./configs/sh-scripts/initDefaultValues.sh && ./configs/sh-scripts/initDevfileTests.sh",
-    "test-regression": "./configs/sh-scripts/generateIndex.sh && ./configs/sh-scripts/initDefaultValues.sh npm run lint && npm run tsc && mocha --config dist/configs/mocharc.js"
+    "test-all-devfiles": " ./configs/sh-scripts/initDefaultValues.sh && ./configs/sh-scripts/initDevfileTests.sh",
+    "test-regression": "./configs/sh-scripts/initDefaultValues.sh npm run lint && npm run tsc && mocha --config dist/configs/mocharc.js",
+    "test-userstory": "./configs/sh-scripts/initDefaultValues.sh npm run lint && npm run tsc && export USERSTORY=$USERSTORY && mocha --config dist/configs/mocharc.js",
+    "test-suite": "./configs/sh-scripts/initDefaultValues.sh npm run lint && npm run tsc && -e MOCHA_DIRECTORY=$MOCHA_DIRECTORY && mocha --config dist/configs/mocharc.js"
```

Comment:
Command list in here looks ruther confusing than clear.

What is the difference between `test-userstory`, `test-suite`, `test-regression`, `test-all-devfiles`?
How to run `EmptyWorkspace.spec.ts` test and `QuarkusWorkspace.spec.ts` test?

So that we are going to support few GUI tests only, it would be simpler for usage to have separate command for each test with clear self-explained name:
- `factory-test`
- `refused-oauth-factory-test`
- `no-setup-factory-test`
- `predefined-namespace-test`
- `empty-workspace-sample-test`
- `quarkus-workspace-sample-test`
- `smoke-test`

Reply:
If you want to run one file you can set file name as userstory and use command "test-userstory":
https://github.com/eclipse/che/blob/8ff5ee8d0ecf6205307f8e0790cb71fee155830e/tests/e2e/README.md#custom-launch

"test-all-devfiles": left because it used in another projects as PR tests, better to change it to "test-userstory"

Reply:
Do propose to have "**test-userstory**" command only, with one parameter "**USER_STORY**", which will accept all test file names without `.spec.ts` suffix inside "**specs**" directory and its sub-directories ?

Reply:
I left one script - "test".
npm run test - launches all test files,
export USERSTORY=Quarkus && npm run test - will run one Quarkus.spec.js file.

README.md updated: https://github.com/eclipse/che/blob/86cecc25878339db64a07e8d056506681840ea3f/tests/e2e/README.md#default-launch

"test-all-devfiles": left because it used in another projects as PR check but it can be replaced to "test"

Reply:
looks good
Thanks!